Blish HUD/GameServices/ModuleService.cs

```diff
@@ -300,7 +300,10 @@ public class ModuleService : GameService {
                     Logger.Warn("Failed to load module from path {modulePath}.", ApplicationSettings.Instance.DebugModulePath);
                 }
 
-                debugModule?.TryEnable();
+                if (debugModule != null) {
+                    debugModule.State.Enabled = true;
```

Comment:
This impacts the state even after we're no longer using `--module`, which is a change in behavior we'll just want to keep in mind.

Before: when no longer using `--module`, a module would adhere to the state it was previously in.
After: when no longer using `--module`, the module will be enabled until disabled again.